rmc/utils/mpc.py

```diff
@@ -411,16 +433,21 @@ def prepare_pop_path_ht(
     # If including rescue RMC results in calculations, drop initial-only OE column
     # If not including, drop rescue-included OE column
     context_ht_cols = set(context_ht.row)
+    oe_col = "oe_rescue" if include_rescue else "oe"
+    if oe_col not in context_ht_cols:
```

Comment:
maybe this oe_col check could get tucked into the if statement above? something like
```
rmc_cols = {oe_col, transcript_col}
if not file_exists(context_with_oe_dedup.path) or (len(rmc_cols.intersection(set(context_with_oe_dedup.ht().row))) < len(rmc_cols)):
```

Reply:
Good call

Reply:
I agree with the addition of the above, but I feel like we could keep this if statement in as it would check that the expected output of running `create_context_with_oe` was generated

Reply:
ahh just to double check. works for me!